### PR TITLE
feat(workflow): Update Alert email design

### PR DESCRIPTION
**Pull request by billyvg**
This updates the alert emails to reflect new designs. Still need to add an actual Environment

image

### CODE REVIEW

File: src/sentry/incidents/action_handlers.py [hunk 1]
```diff
@@ -76,14 +82,12 @@ def resolve(self):
...
         for user_id, email in self.get_targets():
-            email_context["unsubscribe_link"] = self.generate_unsubscribe_link(user_id)
             self.build_message(email_context, status, user_id).send_async(to=[email])
 
     def build_message(self, context, status, user_id):
-        context["unsubscribe_link"] = self.generate_unsubscribe_link(user_id)
         display = self.status_display[status]
         return MessageBuilder(
-            subject=u"Incident Alert Rule {} for Project {}".format(display, self.project.slug),
+            subject=u"Alert Rule {} for Project {}".format(display, self.project.slug),
```
Comment (billyvg) [inline src/sentry/incidents/action_handlers.py]: @adhiraj Do you have an idea of how to format the subject?

File: src/sentry/incidents/action_handlers.py [hunk 2]
```diff
@@ -120,12 +131,19 @@ def generate_email_context(self, status):
                 )
             ),
             "incident_name": self.incident.title,
+            # TODO(alerts): Add environment
+            "environment": "All",
```
Comment (billyvg) [inline src/sentry/incidents/action_handlers.py]: Waiting for #16621 
/cc @iProgramStuff

File: src/sentry/incidents/action_handlers.py [hunk 3]
```diff
@@ -120,12 +139,20 @@ def generate_email_context(self, status):
...
+            "time_window": format_duration(alert_rule.time_window),
```
Comment (wedamija) [inline src/sentry/incidents/action_handlers.py]: `time_window` is in minutes. Don't know why I did that, might refactor it at some point

Comment (billyvg) [reply src/sentry/incidents/action_handlers.py]: oops, updated https://github.com/getsentry/sentry/pull/16867 to be in minutes

File: src/sentry/incidents/action_handlers.py [hunk 4]
```diff
@@ -120,12 +139,20 @@ def generate_email_context(self, status):
...
             "query": alert_rule.query,
-            "threshold": trigger.alert_threshold
-            if status == TriggerStatus.ACTIVE
-            else trigger.resolve_threshold,
-            "status": self.status_display[status],
+            "threshold": trigger.alert_threshold if is_active else trigger.resolve_threshold,
+            # if alert threshold and threshold type is above then show '>'
+            # if resolve threshold and threshold type is *BELOW* then show '>'
+            "threshold_direction_string": ">" if show_greater_than_string else "<",
+            "status": self.incident_status[self.incident.status],
```
Comment (wedamija) [inline src/sentry/incidents/action_handlers.py]: You'll need to cast `self.incident.status` as `IncidentStatus`

File: tests/sentry/incidents/test_action_handlers.py [hunk 5]
```diff
@@ -107,7 +107,7 @@ def test(self):
             ],
             "query": action.alert_rule_trigger.alert_rule.query,
             "threshold": action.alert_rule_trigger.alert_threshold,
-            "status": handler.status_display[status],
+            "status": handler.incident_status[incident.status],
```
Comment (wedamija) [inline tests/sentry/incidents/test_action_handlers.py]: Need another cast

**Review by wedamija — APPROVED** [review]

**Pull request merged**